pkg/image/containerdregistry/certs.go

```diff
@@ -0,0 +1,24 @@
+package containerdregistry
+
+import (
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+)
+
+func RootCAs(CaFile string) (*x509.CertPool, error) {
```

Comment:
Nice! but does this need to be tightly coupled to `containerd`?

Reply:
Not necessarily, it's something we can move to `pkg/lib/`